contrib/google.golang.org/grpc/grpc_test.go

```diff
@@ -591,3 +591,79 @@ func TestAnalyticsSettings(t *testing.T) {
 		assertRate(t, mt, 0.23, WithAnalyticsRate(0.23))
 	})
 }
+
+func TestIgnoredMethods(t *testing.T) {
+	assertUnary := func(t *testing.T, mt mocktracer.Tracer, shouldIgnored bool, opts ...InterceptorOption) {
```

Comment:
Instead of doing this, can we maybe have a table-driven test which is something like this:
```go
for _, ignore := range [][]string{
    {},
    {"/grpc.Fixture/Ping"},
    {"/grpc.Fixture/Ping", "/additional/endpoint"},
} {
    // contents of assertUnary function
}
```
Seems a bit more idiomatic. The current way feels a little bit backwards.

Reply:
Since some of tests are not in tdd,  i thought this was the way to write tests in this repo.

Make it TDD in https://github.com/DataDog/dd-trace-go/pull/577/commits/fdfaef47bd4cefb6932e500f1486fba9a2172738 . Thank you